CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ in development
 Added
 ~~~~~
 
+* Orchestra - new StackStorm-native workflow engine. This is currently in **beta**. (new feature)
```

Comment:
This doesn't say that it "will eventually replace ActionChain and Mistral." like the PR says

Reply:
This is deliberate, to avoid confusion from folks who just read the release notes, and might assume that Action Chains + Mistral workflows are no longer supported.

Better to wait until Orchestra gets to GA, and a deprecation timeline is established for ActionChain/Workflows.

Reply:
I updated the description above.